### DISCUSSION TITLE
Local development

### DISCUSSION THREAD

**Discussion by FrankPeters** [lucky-media/formatic#8] [Q&A]
I am trying to assess if we can use Formatic in our existing Statamic project. It already is up and running and heavily customized (database driver for storage, bootstrap 3 frontend). Is there a way to install Formatic for local development purposes?

**Comment by lokmanm** [comment]
Hey @FrankPeters,

Formatic has a very minimal setup and blueprint. Theoretically, it should work with existing setups unless you tinkered with custom forms. 

> I'm not allowed to tell you this, but you can try this if you download the repo and copy-paste stuff to your project. Make sure to also install the dependencies.
